core/store/orm/orm.go

```diff
@@ -995,6 +995,24 @@ func findUser(db *gorm.DB) (user models.User, err error) {
 	return user, db.Preload(clause.Associations).Order("created_at desc").First(&user).Error
 }
 
+// GetUserWebAuthn will return a list of structures representing all enrolled WebAuthn
+// tokens for the user. This list must be used when logging in (for obvious reasons) but
+// must also be used for registration to prevent the user from enrolling the same hardware
+// token multiple times.
+func (orm *ORM) GetUserWebAuthn(user *models.User) ([]models.WebAuthn, error) {
+	var uwas []models.WebAuthn
+	err := orm.DB.Where("email = ?", user.Email).Find(&uwas).Error
```

Comment:
case insensitive?

Reply:
Updated - although I noted that email creation and log in is not case insensitive yet